packages/notification-services-controller/src/NotificationServicesController/NotificationServicesController.test.ts

```diff
@@ -66,95 +70,64 @@ const clearAPICache = () => {
   notificationsConfigCache.clear();
 };
 
-describe('metamask-notifications - constructor()', () => {
-  it('initializes state & override state', () => {
-    const controller1 = new NotificationServicesController({
-      messenger: mockNotificationMessenger().messenger,
-      env: { featureAnnouncements: featureAnnouncementsEnv },
-    });
-    expect(controller1.state).toStrictEqual(defaultState);
-
-    const controller2 = new NotificationServicesController({
-      messenger: mockNotificationMessenger().messenger,
-      env: { featureAnnouncements: featureAnnouncementsEnv },
-      state: {
-        ...defaultState,
-        isFeatureAnnouncementsEnabled: true,
-        isNotificationServicesEnabled: true,
-      },
+describe('NotificationServicesController', () => {
```

Comment:
Should we reformat the tests in another PR? Otherwise this diff is somewhat difficult to read.